src/App.jsx

```diff
@@ -1,15 +1,39 @@
 import React from 'react';
 import './App.scss';
+import Sum from './components/Sum';
 
 const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove comment lines :) */}
+    <p>
+      <Sum
+        a={2}
+        b={3}
+      />
+    </p>
+    <p>
+      <Sum
+        a={-5}
+        b={5}
+      />
+    </p>
+    <p>
+      <Sum
+        a={10}
+        b={0}
+      />
+    </p>
+    <p>
+      <Sum
+        a={0}
+        b={5}
+      />
+    </p>
+    <p>
+      <Sum
+        a={0}
+        b={0}
+      />
```

Comment:
Ми ж через ```ComponentName.defaultProps``` передаємо значення по дефолту, тобто якщо вони не будуть передані в компоненті як атрибут. Тому немає необхідності явно вказувати 0

```suggestion
      <Sum
        a={10}
      />
    </p>
    <p>
      <Sum
        b={5}
      />
    </p>
    <p>
      <Sum
      />
```